test/react-native/features/feature_flags.feature

```diff
@@ -42,3 +42,32 @@ Scenario: Sends no feature flags after clearFeatureFlags()
   And the exception "errorClass" equals "Error"
   And the event "unhandled" is false
   And event 0 has no feature flags
+
+Scenario: Sends JS feature flags in a native crash
+  When I run "FeatureFlagsNativeCrashScenario" and relaunch the app
```

Comment:
Ah, we're going to have a problem here as I have renamed the step on `next` - apologies.  Could you update the integration branch and this PR first?  The step now says "...and relaunch the crashed app".

Reply:
Actually, ignore that, there are most likely other test scenarios that are affected.  Let's merge this and update the integration branch as a separate activity.